examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,94 @@
+#  Cross Validation for Binary Classifier
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_binary_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_binary_2d_linear_labels_train.dat")
```

Comment:
pls use the new API everywhere

`File f_feats_train 0 csv_file(....`

Reply:
Fixed in coming commit